mysql-test/t/percona_userstat_bug728082.test

```diff
@@ -0,0 +1,43 @@
+--source include/have_innodb.inc
+--source include/count_sessions.inc
+
+SET @userstat_old= @@userstat;
+
+CREATE USER mysqltest_1@localhost;
+CREATE USER mysqltest_2@localhost;
+
+SET GLOBAL userstat=ON;
+
+connect (conn1,localhost,mysqltest_1,,);
+connection default;
+
+SELECT CONCURRENT_CONNECTIONS = 1 AS correct FROM INFORMATION_SCHEMA.USER_STATISTICS WHERE USER LIKE 'mysqltest_1';
+
+connect (conn2,localhost,mysqltest_1,,);
+connection default;
+
+SELECT CONCURRENT_CONNECTIONS = 2 AS correct FROM INFORMATION_SCHEMA.USER_STATISTICS WHERE USER LIKE 'mysqltest_1';
+
+disconnect conn2;
+let $count_sessions= 2;
```

Comment:
Please add a comment that connections opened by this test have overlapping lifetimes, hence the direct count_sessions variable manipulation